client/src/App.tsx

```diff
@@ -23,21 +26,23 @@ function App(): JSX.Element {
           <SnackBarProvider>
             <AuthProvider>
               <SocketProvider>
-                <Switch>
-                  <Route exact path="/settings/profile/">
-                    <Profile />
-                  </Route>
-                  <Route exact path="/login" component={Login} />
-                  <Route exact path="/signup" component={Signup} />
-                  <Route exact path="/settings/editProfile" component={EditProfile} />
-                  <Route exact path="/dashboard">
-                    <Dashboard />
-                  </Route>
-                  {/* <Route path="*">
+                <TourProvider steps={steps}>
+                  <Switch>
+                    <Route exact path="/settings/profile/">
+                      <Profile />
+                    </Route>
+                    <Route exact path="/login" component={Login} />
+                    <Route exact path="/signup" component={Signup} />
+                    <Route exact path="/settings/editProfile" component={EditProfile} />
+                    <Route exact path="/dashboard">
+                      <Dashboard />
+                    </Route>
+                    {/* <Route path="*">
                     <Redirect to="/login" />
                   </Route> */}
```

Comment:
We should try to avoid commenting out code that doesn't need to be commented out.